tests/core/conversion/converters/test_batch_norm.cpp

```diff
@@ -12,25 +12,68 @@ TEST(Converters, ATenBatchNormConvertsCorrectly) {
             %3: Float(5:1),
             %4: Float(5:1)):
         %5 : bool = prim::Constant[value=0]()
-        %6 : float = prim::Constant[value=1.0000000000000001e-05]()
-        %7 : float = prim::Constant[value=0.10000000000000001]()
+        %6 : float = prim::Constant[value=0.10000000000000001]()
+        %7 : float = prim::Constant[value=1.0000000000000001e-05]()
         %8 : Tensor = aten::batch_norm(%0, %1, %2, %3, %4, %5, %6, %7, %5)
         return (%8))IR";
 
   auto g = std::make_shared<torch::jit::Graph>();
   torch::jit::parseIR(graph, &*g);
 
-  auto in = at::randint(1, 10, {1, 5, 5, 5}, {at::kCUDA});
+  auto in = at::randint(1, 10, {2, 5, 5, 5}, {at::kCUDA});
   auto gamma = at::randint(1, 10, {5}, {at::kCUDA});
   auto beta = at::randint(1, 10, {5}, {at::kCUDA});
   auto mean = at::randint(1, 10, {5}, {at::kCUDA});
   auto var = at::randint(1, 10, {5}, {at::kCUDA});
 
+  auto trt_in = at::clone(in);
+  auto trt_gamma = at::clone(gamma);
+  auto trt_beta = at::clone(beta);
+  auto trt_mean = at::clone(mean);
+  auto trt_var = at::clone(var);
+
+  auto params = trtorch::core::conversion::get_named_params(g->inputs(), {gamma, beta, mean, var});
+  auto jit_results = trtorch::tests::util::RunGraph(g, params, {in});
+
+  params = trtorch::core::conversion::get_named_params(g->inputs(), {trt_gamma, trt_beta, trt_mean, trt_var});
+  auto trt_results = trtorch::tests::util::RunGraphEngine(g, params, {trt_in});
+
+  ASSERT_TRUE(trtorch::tests::util::almostEqual(jit_results[0], trt_results[0].reshape_as(jit_results[0]), 2e-6));
+}
+
+TEST(Converters, ATenInstanceNormWithConvertsCorrectly) {
```

Comment:
Another good test would be to make sure we are hitting both paths of the instance norm converter